.nuspec/Xamarin.Forms.Maps.nuspec

```diff
@@ -17,7 +17,7 @@
         <dependency id="Xamarin.Forms" version="$version$"/>
       </group>
       <group targetFramework="MonoAndroid81">
-        <dependency id="Xamarin.GooglePlayServices.Maps" version="60.1142.1"/>
+        <dependency id="Xamarin.GooglePlayServices.Maps" version="71.1610.0"/>
```

Comment:
Does the CG need to be bumped also?

https://github.com/xamarin/Xamarin.Forms/blob/87d720c71ec1c8f8fea2daf2105ba4b6689d71f1/Xamarin.Forms.ControlGallery.Android/Xamarin.Forms.ControlGallery.Android.csproj#L333

Reply:
That's probably a good idea... Done!

Although this specific one the GPS.AppIndexing, doesn't seem to be updated on NuGet

https://www.nuget.org/packages/Xamarin.GooglePlayServices.AppIndexing/

> NOTE: This is package only contains type forwarders to the types which now exist in Xamarin.Firebase.AppIndexing and is only available for backwards compatibility purposes.

Reply:
Can we remove it and just add *Xamarin.Firebase.AppIndexing* instead?

Reply:
That was actually already in there 🤦‍♂